.changes/next-release/Feature-1d2c5bab-06bd-4c5e-a795-f18c7fb34912.json

```diff
@@ -1,4 +1,4 @@
 {
     "type": "Feature",
-    "description": "Systems Manager (SSM) Document support"
+    "description": "New Systems Manager (SSM) Document capabilities: SSM Document are now shown in the AWS Explorer. Language Support (auto-completion, validation) for authoring SSM Document files. Local SSM Document can be created from starting templates. SSM Document can be opened from and published to within an account."
```

Comment:
Consider only reflecting the current state of the PR + feature branch. This file can be revised over the course of the feature branch. That way, if you decide to release at any time, the changelog is more accurate.